### ISSUE TITLE
Using debounce from solid-primitives causes warnings

### ISSUE THREAD

**Issue by aminya** [solidjs-community/eslint-plugin-solid#84]
**Describe the bug**

Using `debounce` from `@solid-primitives` causes reactivity warnings:

**To Reproduce**

```
The reactive variable 'onClick' should be used within JSX, a tracked scope (like createEffect), or inside an event handler function, or else changes will be ignored.eslint[solid/reactivity](https://github.com/solidjs-community/eslint-plugin-solid/blob/main/docs/reactivity.md)
```

```tsx
import { debounce } from '@solid-primitives/scheduled';

function MyComponent() {
	const [something, _] = createSignal(true)
	const onClick = doubounce(() => something(), 1000) // causes warnings
}
```

**Expected behavior**
No warnings

**Screenshots**

**Environment (please complete the following information):**

- OS: [e.g. Mac OS 11, Windows 10]
- Node version (`node --version`): 18.14.2
- `eslint-plugin-solid` version (`npm list eslint-plugin-solid`/`yarn why eslint-plugin-solid`): 0.11.0
- `eslint` version (`npm list eslint`/`yarn why eslint`): 8.35

**Additional context**

- [ ] I would be willing to contribute a PR to fix this issue

**Comment by aminya** [comment]
Is this a real warning or a false positive?

**Comment by ryansolid** [comment]
Closing as covered: add `debounce` (and friends) to the `customReactiveFunctions` option — `{ "solid/reactivity": ["warn", { "customReactiveFunctions": ["debounce", "throttle"] }] }`. We deliberately don't hardcode third-party primitive libraries into the rule; regex support for the option is tracked in #176.
